packages/transport/src/modules/devices/DeviceSecretController.ts

```diff
@@ -89,15 +79,15 @@ export class DeviceSecretController extends TransportController {
         const secretObj = await this.secrets.get(name);
         if (!secretObj) return;
 
-        const baseKey: CryptoSecretKey = this.getBaseKey();
-        const secret: SecretContainerCipher = SecretContainerCipher.from(secretObj);
-        const decryptionKey: CryptoSecretKey = await CoreCrypto.deriveKeyFromBase(baseKey, 1, DeviceSecretController.secretContext);
-        const plainBuffer: ICoreBuffer = await CoreCrypto.decrypt(secret.cipher, decryptionKey);
```

Comment:
It may not be a problem, but when I hover the cursor over the `plainBuffer` variable, I no longer see that it is an `ICoreBuffer` but a `CoreBuffer`.

Reply:
CoreBuffer is way stricter, should be safer now.